weaviate/connect/connection.py

```diff
@@ -240,6 +240,7 @@ def delete(
         self,
         path: str,
         weaviate_object: dict = None,
+        params: dict = None,
```

Comment:
This type should be Optional[Dict[key_type, value_type]] (everywhere)

Reply:
The reason for `params: dict = None` is because it is consistent with existing code.
Nevertheless I changed it to `Optional[dict[str, str]]` like you suggested.

Reply:
I know, there is a lot of old typing that needs to be cleaned up at some point. Btw it is "Dict" (large D) from the typings package

Reply:
Thanks 👍 I change it now to `Optional[Dict[str, Any]]`